### PR TITLE
Add Remembrance, A simple reminder app for Linux

### PR DESCRIPTION
**Pull request by dgsasha**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: com.github.dgsasha.Remembrance.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+app-id: com.github.dgsasha.Remembrance
```
Comment (hfiguiere) [inline com.github.dgsasha.Remembrance.yml:1]: it should be `io.github`

File: com.github.dgsasha.Remembrance.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+runtime-version: '43'
+sdk: org.gnome.Sdk
+command: remembrance
+
+finish-args:
+  - --share=ipc
+  - --device=dri
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --talk-name=org.freedesktop.Notifications
```
Comment (hfiguiere) [inline com.github.dgsasha.Remembrance.yml:12]: you shouldn't need that.

File: com.github.dgsasha.Remembrance.yml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  - --system-talk-name=org.freedesktop.login1
+  - --own-name=com.github.dgsasha.Remembrance
+  - --own-name=com.github.dgsasha.Remembrance.Service
```
Comment (hfiguiere) [inline com.github.dgsasha.Remembrance.yml:15]: In theory this is granted by default based on the app-id.

**Comment by dgsasha** [comment]
This app can autostart, run in the background, send notifications, and talk to `org.freedesktop.login1`. All of this is necessary so it can send notifications for reminders, and to be able to do so at the correct time. Other than that it is fairly light on permissions.

**Comment by dgsasha** [comment]
I typed my commend before you responded by the way, I'll take a look at everything you mentioned

**Comment by dgsasha** [comment]
If I change the app id to `io.github.dgsasha.Remembrance`, will I have to open a new pull request or can I keep it on this branch?

**Comment by hfiguiere** [comment]
the branch name is no concern. just the files need to be renamed.

As for noifications, GNotification use portal so you likely don't need the dbus access.

**Comment by hfiguiere** [comment]
bot, build io.github.dgsasha.Remembrance

**Comment by hfiguiere** [comment]
```
builddir/files/share/appdata/io.github.dgsasha.Remembrance.appdata.xml: FAILED:
• tag-missing           : <release> required
Validation of files failed
```

**Comment by hfiguiere** [comment]
bot, build io.github.dgsasha.Remembrance

**Pull request closed without merging**